internal/rpc/jsonrpc/methods.go

```diff
@@ -2220,10 +2220,8 @@ func (s *Server) getMultisigOutInfo(ctx context.Context, icmd interface{}) (inte
 	}
 
 	// Get the list of pubkeys required to sign.
-	_, pubkeyAddrs, _, err := txscript.ExtractPkScriptAddrs(
-		0, p2shOutput.RedeemScript,
-		w.ChainParams(), true) // Yes treasury
-	if err != nil {
+	_, pubkeyAddrs := stdscript.ExtractAddrs(0 /* TODO */, p2shOutput.RedeemScript, w.ChainParams())
+	if len(pubkeyAddrs) == 0 {
 		return nil, err
```

Comment:
oops, fix this too.  This is my taking-a-break checkpoint, so there's lots still wrong.  :)

Reply:
There only case that returned an error was when the script version was non-zero.  This can just be removed as it was possible to have zero addresses with a `nil` error before too.